lib/tasks/db_data_migrate.rake

```diff
@@ -208,6 +208,19 @@ def populate_nil_project_slugs
   puts " #{slug_count} Project slugs populated."
 end
 
+def set_default_storage_provider
+  if StorageProvider.any?
+    if StorageProvider.where(is_default: true).any?
+      $stderr.puts "0 storage_provider default statuses changed"
+    else
+      StorageProvider.first.update(is_default: true)
+      $stderr.puts "first storage_provider changed to default storage_provider"
+    end
+  else
+    $stderr.puts "no storage_providers found"
```

Comment:
If there aren't any storage providers, should an exception be thrown here? Or maybe we need a separate rake task to ensure that the environment is properly setup? (The latter would be a whole separate task.)

Reply:
I think we need some better post-deployment monitoring, maybe with a properly functioning `/app/status` endpoint. This is something we could remedy without needing to rerun the entire unit-test suite